unblob/cli.py

```diff
@@ -265,19 +265,20 @@ def __init__(
     expose_value=False,
 )
 def cli(
+    *,
```

Comment:
What does this means ?

Reply:
It means, that the rest of the arguments are keyword only arguments: must be supplied with a name.

The problem with parameters of generic types is, that at the call site it can be confusing, and not easily understandable without looking at the function signature.  Boolean and optional parameters are the most confusing.

`f(True, False, None, True)` would hard to understand without knowing the parameter names, which hopefully are self-describing.